packages/suite/src/components/suite/Modal/RawRenderer.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
This should be moved to components package. 

I think we should start moving these dumb components out of suite core folder, especially when it not depends on anything from suite.

Reply:
Normally I would agree, but I am somewhat hoping that we'll get rid of this component completely in the near future, as I really don't like it.

> Suggested future steps:
>
> - complete removal of RawRenderer (nested modal rendering)

Reply:
OK, I only hope it will be near future :D

Reply:
:crossed_fingers: